### PR TITLE
More unsupported browser fixes

### PR DESCRIPTION
**Pull request by samthor**
The `WebComponents.root` var wasn't being passed with a leading slash. But I realised we can restructure our configs to be a bit simpler too.

This also adds a notice informing users that Measure might be unsupported on old browsers.

### CODE REVIEW

**Comment by samthor** [comment]
There's some other JS fixes for Edge.

File: src/site/content/en/measure/index.njk [hunk 1]
```diff
@@ -22,6 +22,9 @@ to remove them as part of the v1 migration. #}
         </span>
       </p>
       <web-signin-button></web-signin-button>
+      <p class="w-page-header__copy w-unsupported-notice">
+        Measure isn't available on your browser. Try a supported browser.
```
Comment (robdodson) [inline src/site/content/en/measure/index.njk]: I can already feel the grumpy tweets 😁

Maybe tweak the language to say. "Sorry, the measure tool does not work with legacy browsers. Please try again using the latest Firefox, Safari, or Chrome."

File: src/styles/components/_page-header.scss [hunk 2]
```diff
@@ -41,3 +41,17 @@
...
+body:not(.unresolved) .w-unsupported-notice {
+  display: none !important;
+}
+
+@keyframes unsupported-appear {
+  from { visibility: hidden; }
+  to   { visibility: hidden; }
+}
+
+.w-unsupported-notice {
```
Comment (robdodson) [inline src/styles/components/_page-header.scss]: Is it worth fading this out or should it just be persistent?

Comment (samthor) [reply src/styles/components/_page-header.scss]: I think it's fine. It only shows up after 5 seconds anyway and we don't expect many users to see it.

File: src/site/content/en/measure/index.njk (hunk 1 above)
Comment (samthor) [reply src/site/content/en/measure/index.njk]: Sounds good.

**Pull request merged**